downstream/modules/platform/con-gw-dash-components.adoc

```diff
@@ -0,0 +1,15 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="gw-dash-components_{context}"]
+
+= {PlatformNameShort} dashboard components
+
+Quick starts:: You can learn about Ansible automation functions with guided tutorials called quick starts. In the dashboard, you can access quick starts by selecting a quick start card. From the panel displayed, click btn:[Start] and complete the onscreen instructions. You can also filter quick starts by keyword and status.
+Resource status:: Indicates the status of your hosts, projects and inventories. The status indicator links to your configured hosts, projects and inventories where you can search, filter, add and modify these resources.
+Job Activity:: You can view a summary of your current job status, filter the job status within a period of time or by job type, or click *Go to Jobs* to view a complete list of jobs that are currently available.
+Jobs:: You can view recent jobs that have run, or click *View all Jobs* to view a complete list of jobs that are currently available, or create a new job.
+Projects: You can view recently updated projects or click *View all Projects* to view a complete list of the projects that are currently available, or create a new project.
```

Comment:
"Projects" has just one colon so it's not being recognized as its own definition.